haiku-apps/haikuwebsearch/haikuwebsearch-0.0.0~git.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A search application"
+DESCRIPTION="A Haiku Application that provides a replicant to search the internet."
+HOMEPAGE="https://github.com/Oyami-Srk/HaikuWebSearch"
+COPYRIGHT="2017 Shiroko"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="c9bc31b35f066870cc73bca78c3f02ac331b1d3f"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="86ccf3e816fac123ac156bb54678001506ddb59e292c3b2a3f4ea36fd9719de1"
+SOURCE_FILENAME="HaikuWebSearch-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="HaikuWebSearch-$scrGitRev"
```

Comment:
```diff
-SOURCE_DIR="HaikuWebSearch-$scrGitRev"
+SOURCE_DIR="HaikuWebSearch-$srcGitRev"
```
(scr -> src)

BTW, I would recommend a rename of the recipe to `haikuwebsearch-0~git.recipe`.

Reply:
I noticed this a few hours ago but forgot to fix lol